pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -79,7 +78,7 @@ func unescapeSystemdCgroupName(part string) string {
 // For example, the name {"kubepods", "burstable", "pod1234-abcd-5678-efgh"} becomes
 // "/kubepods.slice/kubepods-burstable.slice/kubepods-burstable-pod1234_abcd_5678_efgh.slice"
 // This function always expands the systemd name into the cgroupfs form. If only
-// the last part is needed, use path.Base(...) on it to discard the rest.
+// the last part is needed, use filepath.Base(...) on it to discard the rest.
```

Comment:
I agree with the general intent of the patch, but I wonder if we should use OS-independent functions in a OS-dependent source code.

Reply:
Fair enough, will ignore these for the Linux only code.